File: backend/Tips/Dtos/MatchDto.cs
using System;

namespace Tips.Dtos
{
  public class MatchDto
  {
    public long Id { get; set; }
    public byte[] Datum { private get; set; }
    private DateTime Date => new DateTime().AddTicks(BitConverter.ToInt64(Datum, 0));
    public string DateString => $"{Date:dd.MM.yyyy}";
    //public long DateTicks => BitConverter.ToInt64(Datum, 0);
    public string Gruppe { get; set; }
    public long Seq { get; set; }
    public long? Shot { get; set; }
    public long? Received { get; set; }
    public long Team1Id { get; set; }
    public long Team2Id { get; set; }
    public string Team1Name { get; set; }
    public string Team2Name { get; set; }
  }
}

File: frontend/src/models/matchdto.ts
export interface MatchDto {
  id: number;
  dateString: string;
  gruppe: string;
  seq: number;
  shot: number;
  received: number;
  team1Id: number;
  team2Id: number;
  team1Name: string;
  team2Name: string;
}

export const matchDtoDefault: MatchDto = {
  id: -1,
  dateString: '',
  gruppe: 'A',
  seq: 0,
  shot: 0,
  received: 0,
  team1Id: 0,
  team2Id: 0,
  team1Name: '',
  team2Name: '',
};